tests/countries/test_cocos_islands.py

```diff
@@ -0,0 +1,291 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.cocos_islands import CocosIslands, CC, CCK
+from tests.common import CommonCountryTests
+
+
+class TestCocosIslands(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1985, 2050)
+        super().setUpClass(CocosIslands, years=years, years_non_observed=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(CocosIslands, CC, CCK)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(CocosIslands(years=1984))
+
```

Comment:
```suggestion

    def test_special_holidays(self):
        self.assertHoliday(
            "2022-09-22",
        )
```
Standard test for special holidays (instead of `test_national_day_of_mourning_for_queen_elizabeth_ii`).